src/libraries/System.Text.Json/src/System/Text/Json/Reader/JsonReaderHelper.net8.cs

```diff
@@ -0,0 +1,25 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Buffers;
+using System.Runtime.CompilerServices;
+
+namespace System.Text.Json
+{
+    internal static partial class JsonReaderHelper
+    {
+        /// <summary>'"', '\',  or any control characters (i.e. 0 to 31).</summary>
+        /// <remarks>https://tools.ietf.org/html/rfc8259</remarks>
+        private static readonly IndexOfAnyValues<byte> s_controlQuoteBackslash = IndexOfAnyValues.Create(
+            // Any Control, < 32 (' ')
+            "\u0000\u0001\u0002\u0003\u0004\u0005\u0006\u0007\u0008\u0009\u000A\u000B\u000C\u000D\u000E\u000F\u0010\u0011\u0012\u0013\u0014\u0015\u0016\u0017\u0018\u0019\u001A\u001B\u001C\u001D\u001E\u001F"u8 +
```

Comment:
(Implementation detail) IndexOfAny is slightly slower if the values contain `'\0'` on x86.
If we care enough about this case, we could make it a bit cheaper by shifting the values by 1 instead of doing [this](https://github.com/dotnet/runtime/blob/5c9c120bb01d30903890bbec8f0f97b61629945d/src/libraries/System.Private.CoreLib/src/System/IndexOfAnyValues/IndexOfAnyAsciiSearcher.cs#L828-L838), as long as the values don't also contain `0x7F`.